search/typo_suggest_engine.go

```diff
@@ -127,6 +179,7 @@ func (e *ESEngine) GetTypoSuggest(query Query, filterIntents []Intent) (null.Str
 			if considerGrammarTextValue {
 				suggested = strings.Replace(query.Term, checkTerm, suggested, -1)
 			}
+			suggested = queryFormatter.ToResponse(suggested)
```

Comment:
Put this before the "if considerGrammarTextValue" condition
(if considerGrammarTextValue is true, our original checkTerm is not query.Term but the value from the intent).

Reply:
Currently we have panic when checking these terms:

חייים חדשים - 758
חייים חדשים - 810
intoduction to the wisdom item 148
preface to the wisdom pqrt 148

It panics when we run the test_typo_suggest command on these terms.
Make sure test_typo_suggest  is not crashing for English and Hebrew:

./archive-backend test_typo_suggest --eval_set=data/search/en.recall.csv --typos_path=data/search/typos/en.txt --lang=en

./archive-backend test_typo_suggest --eval_set=data/search/he.recall.csv --typos_path=data/search/typos/he.txt --lang=he